pocs/dome/abstract_serial_dome.py

```diff
@@ -0,0 +1,82 @@
+from pocs import dome
+from pocs.utils import error
+from pocs.utils import rs232
+
+
+class AbstractSerialDome(dome.AbstractDome):
+    """Abstract base class for controlling a dome via a serial connection.
+
+    Takes care of a single thing: configuring the connection to the device.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize an AbstractSerialDome.
+
+        Creates a serial connection to the port indicated in the config.
+        """
+        super().__init__(*args, **kwargs)
+
+        # Get config info, e.g. which port (e.g. /dev/ttyUSB123) should we use?
+        # TODO(jamessynge): Switch to passing configuration of serial port in as a sub-section
+        # of the dome config in the YAML. That way we don't intermingle serial settings and
+        # any other settings required.
+        cfg = self._dome_config
+        self._port = cfg.get('port')
+        if not self._port:
+            msg = 'No port specified in the config for dome: {}'.format(cfg)
+            self.logger.error(msg)
+            raise error.DomeNotFound(msg=msg)
+
+        baudrate = int(cfg.get('baudrate', 9600))
+
+        # Setup our serial connection to the given port.
+        self.ser = None
```

Comment:
Only other comment was about turning this into `self.serial` so we have `self.serial.ser` instead of `self.ser.ser`, which is more consistent with the mount. Any reason you want to keep it this way?